app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/covidtests/SingleTestResultFragment.kt

```diff
@@ -246,11 +282,11 @@ class SingleTestResultFragment : Fragment(R.layout.fragment_single_test_result)
     companion object {
 
         @JvmStatic
-        fun newInstance(testRecordDto: TestRecordDto, patientDto: PatientDto) =
+        fun newInstance(testRecordId: String, testResultId: Long) =
```

Comment:
testRecordId must be long

Reply:
using getPatientWithTestResultAndRecords to get the data